game_python/GameLogic.py

```diff
@@ -22,10 +22,10 @@ class GameLogic:
         {"image": (50, 50)},
         {"image": (50, 200)},
         {"image": (50, 350)},
-        {"image": (WINDOW_WIDTH-150, 50)},
-        {"image": (WINDOW_WIDTH-150, 200)},
-        {"image": (WINDOW_WIDTH-150, 350)},
-        {"image": (350, 450)}
+        {"image": (WINDOW_WIDTH - 150, 50)},
+        {"image": (WINDOW_WIDTH - 150, 200)},
+        {"image": (WINDOW_WIDTH - 150, 350)},
+        {"image": (350, 450)},
```

Comment:
czy ta zmiana jest konieczna? zgodnie z PEP 8 - Style Guide for Python operatorów binarnych z największym pryjorytetem nie rozdzielamy.